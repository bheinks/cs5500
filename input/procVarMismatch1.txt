(* 
    Procedure/variable mismatch
*)

program p;

var x : integer;

procedure proc;
begin
  write('p','r','o','c')
end;

begin
  x := proc;
  write('!')
end.

