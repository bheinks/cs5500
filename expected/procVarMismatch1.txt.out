Line 15: Procedure/variable mismatch
